setup.py

```diff
@@ -119,11 +119,11 @@
         "eth-utils>=2.0.0,<3",
         "hexbytes>=0.2.3,<1",
         "py-geth>=3.13.0,<4",
-        "web3[tester]>=6.5.0,<7",
+        "web3[tester]>=6.6.1,<7",
         # ** Dependencies maintained by ApeWorX **
         "eip712>=0.2.1,<0.3",
         "ethpm-types>=0.5.3,<0.6",
-        "evm-trace>=0.1.0a21",
+        "evm-trace==0.1.0a21",
```

Comment:
once the new py-evm, eth-tester, and web3 are all released, we can fix this, as well officially bump mypy in the 1.0 range, holy moly! This is the best we can do for now - it allows it work with both envs if you know what you are doing but CI won't use 1.0 yet in its checks. Locally, both versions of mypy are happy (along with the changes in this PR). Sorry, it got a little whack, but we have some projects using > 1.0 and some are not and everything needs Ape in some capacity